### DISCUSSION TITLE
Some mod multipliers not applied on leaderboard but applied at the same time?

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#22088] [Q&A] [answered]
I've notice that specifically the hold off mod for mania seems to have some weird things going on with the 0.9x multiplier, it shows the score with the multiplier applied, but to beat that score you seem to have to beat the score without the multiplier being applied.
![image](https://user-images.githubusercontent.com/31819497/211264766-a568e96a-cc5a-46b3-ba2a-1dddfbc8ccc2.png)
Screenshot shows the first time where i noticed someone else with the mod being placed higher than he should (until now i only noticed it when i try to beat my scores that i originally set with the mod when playing without the mod, always having to beat the unnerfed score to update the submitted score), to me it looks like the multiplier isn't considered for the placement on the leaderboard for some odd reason.

**Comment by peppy** [comment, marked as answer]
This might be due to the score being old, and the multiplier being different at that point in time. We'll be doing a score reset which should resolve this.
